### PR TITLE
Remove default git info file

### PR DESCRIPTION
**Pull request by tej99**
Remove default git_info file just incase it makes it's way to deployed environments.

### CODE REVIEW

File: tests/Resources/test_info.py [hunk 1]
```diff
@@ -1,4 +1,8 @@
+import os
 import unittest
+from pathlib import Path
+
+from flask import json
```
Comment (andrewmil) [inline tests/Resources/test_info.py]: we should probably just use the standard library json import here for consistency

File: .gitignore [hunk 2]
```diff
@@ -94,4 +94,7 @@ ENV/
 
 # Rope project settings
 .ropeproject
-.build
\ No newline at end of file
+.build
+
+# Test generated files
+git_info
```
Comment (rhiGriff) [inline .gitignore]: No new line at end of file

Comment (tej99) [reply .gitignore]: didn't think that was a think for .gitignore files

Comment (rhiGriff) [reply .gitignore]: I noticed the error on here and thought it was I could be wrong

**Pull request merged**